src/Model/Magazzino.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace MvLabs\Chocosite\Model;
+
+// clausola use per recuperare la classe Prodotto nel namespace corretto
+use MvLabs\Chocosite\Entity\Prodotto;
+
+class Magazzino {
+    private $prodottiMagazzino = [];
+
+    public function __construct($prodottiMagazzino = null) {
+        $this->prodottiMagazzino = $prodottiMagazzino;
+    }
+
+    public function getProdottiMagazzino()
+    {
+        return $this->prodottiMagazzino;
+    }
+
+    public function getDisponibilita($codice)
+    {
+        foreach($prodottiMagazzino as $prodottoMagazzino) {
```

Comment:
non è proprio l'ideale scorrere ogni volta tutti prodotti per recuperarne uno. Per migliorare la cosa avrebbe forse più senso immagazzinare i dati in maniera più facile da recuperare...